importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -0,0 +1,239 @@
+                .start(customTypeImportStep)
+                .next(channelImportStep)
+                .next(createInventoryEntryStep)
+                .next(setPricesStep)
+                .build();
+    }
+
+    @Bean
+    public Step customTypeImportStep(final BlockingSphereClient sphereClient,
+                                     final ItemReader<TypeDraft> customTypeReader,
+                                     final ItemWriter<TypeDraft> customTypeItemWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("customTypeImportStep");
+        return stepBuilder
+                .<TypeDraft, TypeDraft>chunk(1)
+                .reader(customTypeReader)
+                .writer(customTypeWriter(sphereClient))
+                .build();
+    }
+
+    @Bean
+    public Step channelImportStep(final BlockingSphereClient sphereClient,
+                                  final ItemReader<ChannelDraftDsl> channelsDraftReader,
+                                  final ItemWriter<ChannelDraftDsl> channelsDraftItemWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("channelImportStep");
+        return stepBuilder
+                .<ChannelDraftDsl, ChannelDraftDsl>chunk(1)
+                .reader(channelsDraftReader)
+                .writer(channelsDraftWriter(sphereClient))
+                .build();
+    }
+
+    @Bean
+    public Step createInventoryEntryStep(final BlockingSphereClient sphereClient,
+                                         final ItemProcessor<Product, List<InventoryEntryDraft>> inventoryEntryProcessor,
+                                         final ItemWriter<List<InventoryEntryDraft>> inventoryEntryWriter
+    ) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("createInventoryEntryStep");
+        return stepBuilder
+                .<Product, List<InventoryEntryDraft>>chunk(productsImportStepChunkSize)
+                .reader(ItemReaderFactory.sortedByIdQueryReader(sphereClient, ProductQuery.of()))
+                .processor(inventoryEntryProcessor(sphereClient))
+                .writer(inventoryEntryWriter)
+                .build();
+    }
+
+    @Bean
+    public Step setPricesStep(final BlockingSphereClient sphereClient,
+                              final ItemWriter<ProductUpdateCommand> setPriceWriter
+    ) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("createInventoryEntryStep");
+        return stepBuilder
+                .<Product, ProductUpdateCommand>chunk(productsImportStepChunkSize)
+                .reader(ItemReaderFactory.sortedByIdQueryReader(sphereClient, ProductQuery.of()))
+                .processor(setPriceProcessor(sphereClient))
+                .writer(setPriceWriter)
+                .build();
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<TypeDraft> customTypeReader(@Value("#{jobParameters['resourceTypes']}") final Resource typesJsonResource) throws IOException {
```

Comment:
now with the CamelCase it looks better, but I thought again about it and it would be better to call it `typesResources` because the word at the end tells what it is really and the words before just specify it deeper.